hyrax/app/assets/javascripts/hyrax/save_work/required_fields.es6

```diff
@@ -7,80 +11,68 @@ export class RequiredFields {
   }
 
   get areComplete() {
-		this.reload()
-    return this.requiredFields.filter((n, elem) => { return this.isValuePresent(elem) } ).length === 0
+    var missing = this.requiredFields.filter((n, elem) => { return this.isRequiredButBlank(elem) } )
+    return missing.length === 0
 	}
 	
-	requiredIf() {
```

Comment:
merge here is a little gross, but basically all the parts of requiredIf are broken up in to two styling methods that get called after we decide what to do.